File: Final_Project/ClientPC/IoT_ClientPC/IoT_ClientPC/Models/RadarPoint.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IoT_ClientPC.Models
{
    public class RadarPoint
    {
        public int Angle { get; set; }
        public int Distance { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}

File: Final_Project/AndroidClient/app/src/main/java/com/example/androidclient/RadarPoint.java
package com.example.androidclient;

public class RadarPoint {
    public int angle;
    public int distance;
    public long createdAt;

    public RadarPoint(int angle, int distance, long createdAt) {
        this.angle = angle;
        this.distance = distance;
        this.createdAt = createdAt;
    }
}